components/playground/playground.go

```diff
@@ -359,7 +360,36 @@ func (p *Playground) handleScaleIn(w io.Writer, pid int) error {
 	return nil
 }
 
-func (p *Playground) sanitizeConfig(boot instance.Config, cfg *instance.Config) {
+func (p *Playground) absDir(dir string) (string, error) {
```

Comment:
could you add some test for this?

Reply:
Fixed in c0de8f1ffeb5affa75bc680814a7a6f8f68a6901